lib/resources/host.rb

```diff
@@ -48,6 +48,8 @@ def initialize(hostname, params = {})
         @host_provider = LinuxHostProvider.new(inspec)
       elsif inspec.os.windows?
         @host_provider = WindowsHostProvider.new(inspec)
+      elsif inspec.os.darwin?
```

Comment:
Also I kinda feel like you shouldn't really be checking for UDP remotely.. just going to lead to confusing results half the time.